rewrite-weblogic/docs/recipes/migrate-to-weblogic-151100.md

```diff
@@ -75,6 +78,9 @@ dependencies {
 ```
 2. Run `gradle rewriteRun` to run the recipe.
 
+> [!TIP]  
+> If you just want to dry run the recipe without changing the code, use `gradle rewrite:DryRun`.

```

Comment:
Same: The correct command is `gradle rewriteDryRun`. Reference: https://docs.openrewrite.org/reference/gradle-plugin-configuration#the-dryrun-task

Reply:
Should we somewhere put a reference for https://docs.openrewrite.org/reference/gradle-plugin-configuration#the-dryrun-task ?

Reply:
Yes, good idea. I'll add it, then merge the PR.